recipe/meta.yaml

```diff
@@ -23,12 +23,11 @@ requirements:
     - pip
     - cython
     - numpy
-    - libgdal
-    - proj4 <6
+    - libgdal <3
```

Comment:
If you want to merge this just change `<3` to `2.4` and re-render. I guess that having `proj 6` is better than `gdal 3` at this point.

Reply:
Sounds good. Will do. Thanks!